apps/screener-proxy/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "screener-proxy",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "build": "tsc",
+    "start": "node dist/index",
+    "dev": "nodemon index.ts"
+  },
+  "dependencies": {
+    "@octokit/app": "10.1.0",
+    "body-parser": "^1.19.0",
+    "express": "4.17.1"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.11",
+    "@types/node": "^14.14.27",
+    "dotenv": "^8.2.0",
+    "ngrok": "^3.4.0",
```

Comment:
Remove this since it appears unused, and if at all possible we should avoid installing extra versions of ngrok since it has a slow postinstall script